Jenkinsfile

```diff
@@ -57,49 +57,12 @@ node ('master') {
         }
 
         // Run the tests
-        stage("Run 0-8 Tests"){
+        stage("Run Tests") {
             // Required docker containers are built by the tests
             sh './bin/docker_build_all -p $(printf $BUILD_TAG | sha256sum | cut -c1-64)'
             sh './bin/run_tests -p $(printf $BUILD_TAG | sha256sum | cut -c1-64)'
         }
 
-        stage("Remove Docker Images") {
```

Comment:
I think so, as a similar stage "Remove docker images" removes both the build and docs images.